Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/SingleFileTransfer.cs

```diff
@@ -456,7 +457,10 @@ private string RestoreSourceFileAfterErrorIfItWasRenamed()
             {
```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/33bbc16e411f6fa4f5f8086d85e2a83889335206/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/SingleFileTransfer.cs#L445
This can be a static method.